src/index.js

```diff
@@ -0,0 +1,8 @@
+import { displayCatName } from './cat_generator';
+
+(() => {
```

Comment:
Look at this super-fancy ES6 stuff. I see why you wanted to verify browser requirements